ekncontent/ekncontent/eknc-database-manager.c

```diff
@@ -669,14 +669,14 @@ parse_query_flags (const gchar *str,
 
 static gboolean
 eknc_database_manager_fix_query_internal (EkncDatabaseManager *self,
+                                          const char *query_str,
```

Comment:
This is an artefact caused by rebasing on master; it doesn't make any difference, because the quest string is retrieved exactly the same way on both ends.

Originally, I moved the query string out of the hash table and into an argument because I was passing different queries with the same parameters; but now that the query string is one, it's less useful, though a fairly logical clean up.

For SDK3 I want to nuke the parameters hash table, and just pass the QueryObject instance around.